nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -415,3 +405,17 @@ def unpatch_torch_operators():
 
     for orig_op_info in ORIGINAL_OPERATORS:
         setattr(orig_op_info.namespace, orig_op_info.name, orig_op_info.op)
+
+
+@contextmanager
+def disable():
```

Comment:
The name is kind of ambiguous, moreover if we make it available from `nncf.torch`. I suggest `patching_disabled` or something like that.

Reply:
Renamed. Note that this context manager is not yet part of the API (although I imagine it will be in the future) since it is not decorated as such via `@api`.